src/cli/study-backrefs.js

```diff
@@ -0,0 +1,358 @@
+  "content-security-policy": "CSP",
+  "feature-policy": "permissions-policy",
+  "css2": "CSS21",
+  "css-contain-1": "css-contain",
+  "css3-background": "css-backgrounds",
+  "css3-break": "css-break",
+  "css3-color": "css-color",
+  "css-color-3": "css-color",
+  "css3-align": "css-align",
+  "css3-box": "css-box",
+  "css3-flexbox": "css-flexbox",
+  "css3-fonts": "css-fonts",
+  "css-fonts-3": "css-fonts",
+  "css3-grid-layout": "css-grid",
+  "css-grid-1": "css-grid",
+  "css3-animations": "css-animations",
+  "css3-images": "css-images",
+  "css3-mediaqueries": "mediaqueries",
+  "css3-multicol": "css-multicol",
+  "css3-namespace": "css-namespaces",
+  "css3-page": "css-page",
+  "css3-positioning": "css-position",
+  "css3-regions": "css-regions",
+  "css3-speech": "css-speech",
+  "css3-syntax": "css-syntax",
+  "css3-transforms": "css-transforms",
+  "css-ui-3": "css-ui",
+  "css3-text": "css-text",
+  "css3-transitions": "css-transitions",
+  "css3-values": "css-values",
+  "css3-writing-modes": "css-writing-modes",
+  "css-writing-modes-3": "css-writing-modes",
+  "css3-selectors": "selectors",
+  "css-selectors-3": "selectors",
+  "selectors-3": "selectors",
+  "css-selectors": "selectors",
+  "selectors4": "selectors",
+  "webdriver1": "webdriver2",
+  "webdriver": "webdriver2",
+  "resource-timing": "resource-timing-2",
+  "html-aam": "html-aam-1.0",
+  "ServiceWorker": "service-workers",
+  "BackgroundSync": "background-sync",
+  "InputDeviceCapabilities": "input-device-capabilities",
+  "IntersectionObserver": "intersection-observer",
+  "wasm-core-1": "wasm-core",
+  "pointerevents2": "pointerevents",
+  "input-events-1": "input-events",
+  "accname-aam-1.1": "accname",
+  "core-aam-1.1": "core-aam",
+  "accname-1.1": "accname",
+  "webauthn-1": "webauthn",
+  "resource-timing-1": "resource-timing",
+  "wai-aria-1.1": "wai-aria-1.2"
+};
+
+// TODO: check the link is non-normative (somehow)
+const shortnameOfNonNormativeDocs = [
+  "aria-practices", "rdf11-primer", "discovery-api", "wake-lock-use-cases", "capability-urls", "streamproc", "media-source-testcoverage", "using-aria", "wai-aria-practices-1.1", "sensor-polyfills", "sensors", "design-principles", "security-questionnaire", "css3-preslev", "clreq", "klreq", "typography", "ssml-sayas", "css3-marquee", "spatial-navigation", "ilreq", "css-print", "books", "dpub-pagination", "predefined-counter-styles", "css-2017", "sniffly", "wai-aria-practices-1.2", "wai-aria-implementation", "wai-aria-roadmap", "wai-aria-practices", "webdatabase", "installable-webapps", "motion-sensors", "file-system-api", "media-accessibility-reqs", "webrtc-interop-reports", "webaudio-usecases", "web-audio-perf", "Audio-EQ-Cookbook", "web-intents", "touch-events-extensions", "fingerprinting-guidance", "webrtc-nv-use-cases", "html-design-principles", "storage-partitioning", "jlreq", "accept-encoding-range-test", "security-privacy-questionnaire", "dpub-latinreq"
+];
+
+const report = {};
```

Comment:
It would be better not to create global variables and rather move that within `studyCrawlResults`, in case we make this a real module later on.